PURL-TYPES.rst

```diff
@@ -59,6 +59,24 @@ apk
       pkg:apk/alpine/curl@7.83.0-r0?arch=x86
       pkg:apk/alpine/apk@2.12.9-r3?arch=x86
 
+bazel
+-----
+``bazel`` for Bazel modules:
+
+- The default repository ("registry") is ``https://bcr.bazel.build``, the
+  Bazel Central Registry (BCR).
+- The ``namespace``, if specified, is the host and path of the URL of an
+  alternative registry. It is not case sensitive and must be lowercased.
```

Comment:
The "must be lowercased" part does not match the generic instructions [how to build a purl](https://github.com/package-url/purl-spec/blob/master/PURL-SPECIFICATION.rst#how-to-build-purl-string-from-its-components), so I think it should be dropped here. IMO it's not feasible for implementers to add special handling for Bazel here.

Reply:
Ah, my bad, for some reason I seem to have stopped reviewing at the first commit. Anyway, good to have the fixups squashed now.